evap/static/scss/components/_callouts.scss

```diff
@@ -94,10 +94,19 @@
 }
 
 .infobox.callout {
-  max-width: 100%;
+    min-width: 100%;
 
-  // enlarge infobox to max size
-  .callout-content {
-    width: 1300px;
-  }
+    &.closing,
+    &.closed {
+        min-width: 0;
+    }
+
+    // override transitions to include min-width
+    &.closing {
+        transition: border-radius 1s ease 1s, border-left-width 1s ease 1s, padding 1s ease-in-out, min-width 1s ease 1s;
+    }
+
+    &.opening {
+        transition: border-radius 1s ease, border-left-width 1s ease, padding 1s ease-in-out, min-width 1s ease;
+    }
```

Comment:
This looks like the other transition attributes are copied from somewhere, so we now have an invisible dependency here, and if we change the other place, we'd need to change this here too, which we'd very likely forget. Is that what's happening here?

Reply:
Yes, these transitions are the same from l. 80 & l. 88, and add the min-width property. We can probably create a mixin to reduce duplication.
In general, I tried not to edit ll. 1-94, as the code for these callouts seem to be copied from some github gist